ydb/core/persqueue/partition.cpp

```diff
@@ -933,6 +933,25 @@ void TPartition::Handle(TEvPQ::TEvTxRollback::TPtr& ev, const TActorContext& ctx
     ContinueProcessTxsAndUserActs(ctx);
 }
 
+void TPartition::Handle(TEvPQ::TEvGetWriteInfoRequest::TPtr& ev, const TActorContext& ctx) {
+    if (CurrentStateFunc() != &TThis::StateIdle || !Requests.empty()) {
+        auto* response = new TEvPQ::TEvGetWriteInfoError(Partition.InternalPartitionId,
+                                                       "Write info requested while writes are not complete");
+        ctx.Send(ev->Sender, response);
+        return;
+    }
+    ClosedInternalPartition = true;
```

Comment:
Check this variable in every write, ownership or reserve bytes request and answer error if ClosedInternalPartition == true.